src/components/tests/assistant-anchor.test.tsx

```diff
@@ -0,0 +1,105 @@
+import * as Sentry from '@sentry/electron/renderer';
+import { render, screen } from '@testing-library/react';
+import { speak } from '@wordpress/a11y';
+import { useSiteDetails } from '../../hooks/use-site-details';
+import { getIpcApi } from '../../lib/get-ipc-api';
+import Anchor from '../assistant-anchor';
+
+jest.mock( '@sentry/electron/renderer' );
+jest.mock( '../../hooks/use-site-details' );
+jest.mock( '../../lib/get-ipc-api' );
+jest.mock( '@wordpress/a11y' );
+
+describe( 'Anchor', () => {
+	beforeAll( () => {
+		( useSiteDetails as jest.Mock ).mockReturnValue( {} );
+		( getIpcApi as jest.Mock ).mockReturnValue( {
+			openURL: jest.fn( () => Promise.resolve() ),
+			showMessageBox: jest.fn(),
+		} );
+	} );
+
+	it( 'should render an anchor element', () => {
+		render( <Anchor href="https://example.com" children="Example link" /> );
+
+		expect( screen.getByRole( 'link' ) ).toBeInTheDocument();
```

Comment:
In https://github.com/Automattic/studio/pull/307, I tried to modify all assertions to use `toBeVisible` instead of `toBeInTheDocument`. What do you think about using `toBeVisible` here too?